src/index.js

```diff
@@ -125,11 +115,7 @@ const loadRaw = async function (self, {release, gc, target}, srcDir) {
   const cmd = cargoCommand(target, release);
   const result = await execPermissive(cmd, srcDir);
 
-  let {wasmFile} = await handleCargo(self, result);
-
-  if (!wasmFile) {
```

Comment:
This can still happen even when Cargo doesn't throw an error; for example when the library is not a `"cdylib"` so please keep these lines.